docs/orchestrating-elastic-stack-applications/elasticsearch/es-secure-settings.asciidoc

```diff
@@ -32,10 +32,23 @@ kind: Secret
 metadata:
   name: one-secure-settings-secret
 type: Opaque
-data:
-  gcs.client.default.credentials_file: RWxhc3RpYyBDbG91ZCBvbiBLOHMgKEVDSykK
+stringData:
+  gcs.client.default.credentials_file: |
+    {
+      "type": "service_account",
+      "project_id": "your-project-id",
+      "private_key_id": "...",
+      "private_key": "-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n",
+      "client_email": "service-account-for-your-repository@your-project-id.iam.gserviceaccount.com",
+      "client_id": "...",
+      "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+      "token_uri": "https://accounts.google.com/o/oauth2/token",
+      "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+      "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/your-bucket@your-project-id.iam.gserviceaccount.com"
+    }
 ----
 
+TIP: Note that by default https://kubernetes.io/docs/concepts/configuration/secret/[Kubernetes secrets] are expecting the value to be base64 encoded unless under a `stringData` field. In the previous example,  `RWxhc3RpYyBDbG91ZCBvbiBLOHMgKEVDSykK` base64 decodes to `Elastic Cloud on K8s (ECK)`.
```

Comment:
It seems to me that we have to choose between modifying the example secret in yaml format and adding the tip. As it stands, the string `RWxhc3RpYyBDbG91ZCBvbiBLOHMgKEVDSykK` mentioned in the tip does not exist, it's confusing.

Reply:
Yes let's leave that out.

Reply:
Fixed!